src/amber/controller/filters.cr

```diff
@@ -80,6 +80,12 @@ module Amber::Controller
       filters[filter.precedence] << filter
     end
 
+    def each(&block : {Symbol, Array(Filter)} -> _)
```

Comment:
What was the point of adding this?

Reply:
it's now required to implement abstract methods in Crystal.